externs/browser/webkit_dom.js

```diff
@@ -194,6 +200,12 @@ Console.prototype.profileEnd = function(opt_title) {};
  */
 Console.prototype.time = function(name) {};
 
+/**
+ * @param {string} label
+ * @return {undefined}
+ */
+Console.prototype.timeLog = function(label) {};
```

Comment:
Per the spec, this needs an additional `@param {*...} data`.

https://console.spec.whatwg.org/#timing

Reply:
Interesting - I had not seen that before. It is not documented at https://developer.mozilla.org/en-US/docs/Web/API/Console/timeLog but I will try it out in the browser and if it works add it in.

Reply:
New commit pushed with change